src/containers/ProductListing.js

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect } from 'react';
+import axios from 'axios';
+import { useDispatch, useSelector } from 'react-redux';
+import { setProducts } from '../redux/actions/productsActions';
+import ProductComponent from './ProductComponent';
+
+const URL = 'https://fakestoreapi.com/products';
+
+const ProductPage = () => {
+  const products = useSelector((state) => state.allProducts.products);
+  const dispatch = useDispatch();
+  const getProducts = async () => {
+    const response = await axios
+      .get(URL)
+      .catch((err) => {
+        console.log('Err: ', err);
+      });
+    dispatch(setProducts(response.data));
+  };
+
+  useEffect(() => {
+    getProducts();
+  }, []);
+
+  console.log('Products :', products);
```

Comment:
- [ ] As the previous reviewer stated please follow the instructions below.

 ![image](https://user-images.githubusercontent.com/58553711/203684466-4fbf4886-3b66-4fe1-bb4d-281ea32bec87.png)
> 
 * Your design
  ![image](https://user-images.githubusercontent.com/58553711/203681529-ac2a974b-1c8b-47e2-ae94-216f3451685e.png)
 * Original design
   ![image](https://user-images.githubusercontent.com/58553711/203681651-b72e62dc-126e-4f45-be1f-14eb4fc46166.png)